### PR TITLE
feature: update docs

### PR DESCRIPTION
**Pull request by luislucena16**
i) Done:
- updated all web2 and web3 documentation and added more descriptive comments.
- added `REVIEW` comments to be evaluated with @RichardIrala ..

ii) Under review:
- change the order of the parameters in:
`getListedNftsOnSecondaryMarket`, `getPlayerItemsOnSecondaryMarket`, `secondaryPurchaseWithXerialWallet`
please check this and tell me if everything is OK!!!

- delete a userAddres in:
`getMaticBalance` => are repeated and say the same thing

- the `JSON` response is just the response? without the object name, right? I think yes but I'm not sure!
```json
"0.053353126320360021"
```

cc @RichardIrala

### CODE REVIEW

File: functions/web2/web2.js [hunk 1]
```diff
@@ -247,8 +256,10 @@ web2Functions.getMaticBalance = async function ({ userAddress, chain }) {
...
 };
 
-// web2Functions.transferMatic = async function () {}
-
-// web2Functions.transferUsdc = async function () {}
+// REVIEW: can this be deleted? cc @RichardIrala
+/*
+  // web2Functions.transferMatic = async function () {}
+  // web2Functions.transferUsdc = async function () {}
+*/
```
Comment (RichardIrala) [inline functions/web2/web2.js]: You can delete web2functions.transferUsdc

File: functions/web3/web3.js [hunk 2]
```diff
@@ -6,16 +6,20 @@ import telosGasLimit from "@/utils/telosGasLimit";
 import getEthereumChainParam from "@/utils/getEthereumChainParam";
 import { Decimal } from "decimal.js";
 const ethereum = globalThis.ethereum;
+
+// REVIEW: this is not being called cc @RichardIrala, is this where you will be prompted to install/download MetaMask?
```
Comment (RichardIrala) [inline functions/web3/web3.js]: It will be used at some point, or another developer will use it

File: pages/inventory.js [hunk 3]
```diff
@@ -195,6 +195,7 @@ const Inventory = ({ activeChain, handleActiveChain }) => {
     }
   }
 
+  // REVIEW: some parameters in this function are not read, you will use them later? cc @RichardIrala
```
Comment (RichardIrala) [inline pages/inventory.js]: I will use it later

File: functions/web2/web2.js (hunk 1 above)
Comment (luislucena16) [reply functions/web2/web2.js]: done!

File: functions/web3/web3.js (hunk 2 above)
Comment (luislucena16) [reply functions/web3/web3.js]: ok! it stays!

Comment (luislucena16) [reply functions/web3/web3.js]: done! I delete the comment!

File: pages/inventory.js (hunk 3 above)
Comment (luislucena16) [reply pages/inventory.js]: nice! I delete the comment!

**Review by RichardIrala — APPROVED** [review]

**Pull request merged**